kpatch-build/create-diff-object.c

```diff
@@ -69,6 +69,8 @@
 		printf(format, ##__VA_ARGS__); \
 })
 
+extern char *find_exported_symbol_objname(const char *fname, char *symname);
```

Comment:
License and copyright are two separate things.  The license is the same (GPLv2) so it's ok to copy the code -- you don't need to do anything there because lookup.c already has the license notice.

For the copyright, the original file's copyrights should be preserved.  So you can add something to the existing lookup.c copyright statement like "For functions A and B:" and then copy the original copyright statements.  You can add your own copyright statement at the end if you modify the functions.